src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -39,17 +40,54 @@ import { StyleableRelType } from './StyleableRelType'
 import { upperFirst } from 'services/utils'
 import { ShowMoreOrAll } from 'browser-components/ShowMoreOrAll/ShowMoreOrAll'
 
+export const MAX_LENGTH_SMALL = 150
+export const MAX_LENGTH_LARGE = 300
+type ExpandableValueProps = {
+  value: string
+  type: string
+  wideMode: boolean
+}
+function ExpandableValue({ value, wideMode }: ExpandableValueProps) {
+  const [expanded, setExpanded] = useState(false)
+
+  const maxLength = wideMode ? MAX_LENGTH_LARGE : MAX_LENGTH_SMALL
+
+  const handleExpandClick = () => {
+    setExpanded(true)
+  }
+
+  let valueShown = expanded
+    ? value
+    : value.slice(0, Math.min(maxLength, value.length))
```

Comment:
From testing it doesn't matter if slice is called with a value that's too big, so we could do `value.slice(0, maxLength)`. Doing it that way is a bit simpler but maybe less clear. I'll leave it to your preference to pick

Reply:
sure, since slice is such a common method and it is clear in the documentation of that that it will be the whole string in case the end is larger than the length I think it is better to remove the Math.min for simplicity.